tensorflow/python/BUILD

```diff
@@ -3759,7 +3759,7 @@ cuda_py_test(
     main = "ops/nn_fused_batchnorm_test.py",
     python_version = "PY3",
     shard_count = 24,
-    tags = ["no_rocm"],
+    tags = [],
```

Comment:
we can remove this `tags=[],` as it's not used anymore.

Reply:
done